src/client.cpp

```diff
@@ -98,6 +100,9 @@ Client::Client(Window window, bool visible_already, ClientManager& cm)
         updateEwmhState();
         hook_emit({"fullscreen", fullscreen_() ? "on" : "off", WindowID(window_).str()});
     });
+    sticky_.changed().connect([this] {
```

Comment:
this addition here can be removed, right?

Reply:
Indeed, yes!